### PR TITLE
Upgrade to akka 2.4.18, http 10.0.7, and kamon 0.6.6.

### PR DESCRIPTION
**Pull request by jypma**
This also removes ts-reaktive-kamon-akka-http, since the plain
kamon-akka-http module (0.6.8) has since picked up our code.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -43,19 +45,20 @@ lazy val commonSettings = projectSettings ++ Seq(
...
       "io.kamon" %% "kamon-core" % kamonVersion,
-      "io.kamon" %% "kamon-akka" % kamonVersion,
+      "io.kamon" %% "kamon-akka-2.4" % kamonVersion,
       "io.kamon" %% "kamon-statsd" % kamonVersion,
       "io.kamon" %% "kamon-datadog" % kamonVersion,
       "io.kamon" %% "kamon-log-reporter" % kamonVersion,
       "io.kamon" %% "kamon-system-metrics" % kamonVersion,
       "org.aspectj" % "aspectjweaver" % "1.8.8",
-      "io.kamon" %% "kamon-autoweave" % kamonVersion,
+      "io.kamon" %% "kamon-autoweave" % "0.6.5", // missing for 0.6.6
```
Comment (alar17) [inline build.sbt]: Why it's missing in this version of kamon?!

File: build.sbt [hunk 2]
```diff
@@ -28,11 +28,13 @@ lazy val projectSettings = PB.protobufSettings ++ Seq(
...
 )
 
+lazy val akkaVersion = "2.4.18"
+
 lazy val commonSettings = projectSettings ++ Seq(
   libraryDependencies ++= {
-    val akkaVersion = "2.4.17"
-    val akkaHttpVersion = "10.0.4"
-    val kamonVersion = "0.6.2"
+    val akkaVersion = "2.4.18"
```
Comment (ayurchuk) [inline build.sbt]: Can we keep one `val akkaVersion`?

Comment (jypma) [inline build.sbt]: what do you mean?

File: build.sbt [hunk 3]
```diff
@@ -28,11 +28,13 @@ lazy val projectSettings = PB.protobufSettings ++ Seq(
   )
 )
 
+lazy val akkaVersion = "2.4.18"
```
Comment (alar17) [inline build.sbt]: Why did you define it here as well as in the commonSetting?

File: build.sbt (hunk 1 above)
Comment (jypma) [inline build.sbt]: I'll let you know as soon as I get a reply in https://gitter.im/kamon-io/Kamon :-)

File: build.sbt (hunk 2 above)
Comment (jypma) [inline build.sbt]: ah, I see now. Fixing.

File: build.sbt (hunk 3 above)
Comment (jypma) [inline build.sbt]: Fixed now, only one occurrence of 2.4.18. Also moved `testkit`'s dependency into the root `build.sbt`, so there's really only one reference to the akka version.

**Comment by ayurchuk** [comment]
:star:

**Comment by ayurchuk** [comment]
:star:

**Pull request merged**